v03_pipeline/lib/tasks/write_new_variants_table.py

```diff
@@ -142,31 +148,35 @@ def requires(self) -> list[luigi.Task]:
         return upstream_table_tasks
 
     def complete(self) -> bool:
-        return super().complete() and hl.eval(
-            hl.bind(
-                lambda updates: hl.all(
-                    [
-                        updates.contains(
-                            hl.Struct(
-                                callset=callset_path,
-                                project_guid=project_guid,
-                            ),
-                        )
-                        for (
-                            callset_path,
-                            project_guid,
-                            _,
-                            _,
-                        ) in callset_project_pairs(
-                            self.callset_paths,
-                            self.project_guids,
-                            self.project_remap_paths,
-                            self.project_pedigree_paths,
-                        )
-                    ],
+        return (
+            not self.force
```

Comment:
do we still need this if the new variants table is in the runs directory?

Reply:
Nope!  Good call!